sdk/samples/iot/aziot_esp32/how_to_iot_hub_esp32.md

```diff
@@ -1,3 +1,15 @@
+---
+page_type: sample
+description: Connecting an ESP32 device to Azure IoT using the Azure Embedded C SDK
```

Comment:
```suggestion
description: Connecting an ESP32 device to Azure IoT using the Azure SDK for Embedded C
```

Reply:
I like your better, but this is the title on the first page of the repo...

Reply:
Agreed! Done in the 3rd commit.